server/liveblog/blogs/request_membership.py

```diff
@@ -72,17 +71,19 @@ class MembershipResource(Resource):
         'default_sort': [('_updated', -1)]
     }
     resource_methods = ['POST']
-    privileges = {'POST': 'blogs'}
+    privileges = {'POST': 'request_membership'}
 
 
-class MembershipService(BlogService):
-    notification_key = 'request'
 
+class MembershipService(BaseService):
+    notification_key = 'request'
+  
     def on_create(self, docs):
         for doc in docs:
+            doc['original_creator'] = str(get_user().get('_id'))
             doc['message'] = "Please add me as a contributor to your blog"
```

Comment:
if you are just using that to let front-end know what kind of request it is and we only get one kind of request from that endpoint, I can do without it